Python/dtoa.c

```diff
@@ -697,6 +697,7 @@ pow5mult(Bigint *b, int k)
     for(;;) {
         assert(p5s != interp->dtoa.p5s + Bigint_Pow5size);
         p5 = *p5s;
+        assert(p5 != NULL);
```

Comment:
I'm not sure of the purpose of this assert. If you want to keep it, can you add a comment to explain when the assert can fail?